src/defaultsettings.cpp

```diff
@@ -57,6 +57,10 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("curl_verify_cert", "true");
 	settings->setDefault("enable_remote_media_server", "true");
 	settings->setDefault("enable_client_modding", "false");
+	settings->setDefault("enable_knockback", "true");
+	settings->setDefault("knockback_default_duration", "0.3");
```

Comment:
Yes just one value for knockback strength, it would be the speed that is applied in the direction of hit.